src/executor/LanceurExecutor.java

```diff
@@ -28,11 +32,9 @@ public Integer call() throws Exception {
 		ExecutorService execs = Executors.newSingleThreadExecutor();
 
 		Future<Integer> f = execs.submit(monCallable);
-		System.out.println("suite");
 		try {
 			Integer retour = f.get();
-			System.out.printf("j'ai eu le retour %s%n", retour);
-
+			log.log(Level.INFO, "j'ai eu le retour %s%n", retour);
 		} catch (InterruptedException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2142)